models/biocro/R/model2netcdf.BIOCRO.R

```diff
@@ -78,7 +79,7 @@ model2netcdf.BIOCRO <- function(result, genus = NULL, outdir, lat = -9999, lon =
                        Evap = udunits2::ud.convert(SoilEvaporation + CanopyTrans, "Mg/ha/h", "kg/m2/s"), 
                        TVeg = udunits2::ud.convert(CanopyTrans, "Mg/ha/h", "kg/m2/s"), 
                        LAI = LAI))
-    
+    RR$NPP <- udunits2::ud.convert(c(0, diff(R$TotLivBiom)), "kg/m2/h", "kg/m2/s")
```

Comment:
1) @infotroph wnits need to adhere to the standard -- we can't have different models using different units: http://nacp.ornl.gov/MsTMIP_variables.shtml

2) Note that the standard is kg C m-2 s-1 -- just wanted to double check that the above is C not biomass